ecl/resource_ecl_sss_approval_request_v1.go

```diff
@@ -0,0 +1,182 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/nttcom/eclcloud/ecl/sss/v1/approval_requests"
+)
+
+func resourceSSSApprovalRequestV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSSSApprovalRequestV1Create,
+		Read:   resourceSSSApprovalRequestV1Read,
+		Update: resourceSSSApprovalRequestV1Update,
+		Delete: resourceSSSApprovalRequestV1Delete,
+		Schema: map[string]*schema.Schema{
+			"request_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"external_request_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"approver_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"approver_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"request_user_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"service": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"actions": &schema.Schema{
```

Comment:
Seems like actins is list.
https://ecl.ntt.com/en/documents/api-references/sss/rsts/APIList/Approval/GetApprovalRequest.html#id1

Reply:
that's right.I'll do it.